cpp/src/transform/jit/kernel.cu

```diff
@@ -55,7 +103,25 @@ CUDF_KERNEL void kernel(cudf::size_type size, Out* __restrict__ out, In const* _
   thread_index_type const stride = block_size * gridDim.x;
 
   for (auto i = start; i < static_cast<thread_index_type>(size); i += stride) {
-    GENERIC_TRANSFORM_OP(&out->get(i), ins->get(i)...);
+    GENERIC_TRANSFORM_OP(&Out::get(views, i), In::get(views, i)...);
+  }
+}
+
+template <typename Out, typename... In>
+CUDF_KERNEL void fixed_point_kernel(cudf::size_type size,
+                                    cudf::jit::column_device_view const* views)
+{
+  // cannot use global_thread_id utility due to a JIT build issue by including
```

Comment:
We probably want to make our core utility headers more NVRTC / Jitify friendly. Would it be possible to refactor that header to be supported, by splitting it up into a header with minimal functions and a header with more complex features that Jitify doesn’t support? What changes would be needed?

I would recommend evaluating this now but doing the work in a separate PR.

Reply:
it should be possible (PS: I found that comment in the source code, it's not written by me).
I'll create a follow-up issue for that so we don't forget it.

Reply:
I've created an issue to track this: https://github.com/rapidsai/cudf/issues/18216